app/views/admin/events/index.html.haml

```diff
@@ -25,6 +25,8 @@
         %th
           %b Pre-registration
         %th
+          %b Registrations
+        %th
```

Comment:
@davidsf This change is making [following assertion](https://github.com/openSUSE/osem/blob/master/spec/views/admin/events/index.html.haml_spec.rb#L24) fail:
```Ruby
expect(rendered).to have_selector('table thead th:nth-of-type(6)', text: 'Highlight')
```
Please fix it.